pkg/prometheus/testdata/TargetLimits-1_Versionv2.21.0_Enforce1000.golden

```diff
@@ -66,4 +66,3 @@ scrape_configs:
     - __tmp_hash
     regex: $(SHARD)
     action: keep
-  target_limit: 1000
```

Comment:
hmm we had a bug?

Reply:
@nicolastakashi thoughts?

Reply:
@simonpasquier checking yet, I'll update soon.

Reply:
I suppose that it's the isssue with global sample_limit not being supported before 2.45.